src/adapter/src/catalog.rs

```diff
@@ -1363,6 +1364,15 @@ pub struct ClusterReplica {
 }
 
 impl ClusterReplica {
+    pub fn role(&self) -> ReplicaRole {
+        // NOTE - These roles power monitoring systems. Do not change
+        // them without talking to the cloud or observability groups.
+        match self.name.as_str() {
+            "mz_system" => ReplicaRole::SystemCritical,
+            "mz_introspection" => ReplicaRole::System,
```

Comment:
Is it possible to replace these with `SYSTEM_USER.name` and `INTROSPECTION_USER.name`?

Reply:
Yes, that's a good point